doc-surrealdb_versioned_docs/version-1.x/surrealql/parameters.mdx

```diff
@@ -7,19 +7,51 @@ description: Parameters can be used like variables to store a value which can th
 
 # Parameters
 
-Parameters can be used like variables to store a value which can then be used in a subsequent query. A parameter can store any value, including the result of a query. Parameters can be defined within the SQL, or can be passed in using the client libraries as request variables.
+Parameters can be used like variables to store a value which can then be used in subsequent queries. To define a parameter in SurrealQL, use the [`LET`](../surrealql/statements/let) statement. The name of the parameter should begin with a `$` character.
 
 ## Defining parameters within SurrealQL
 
-To define a parameter in SurrealQL, use the [`LET`](../surrealql/statements/let) statement. The name of the parameter should begin with a `$` character.
-
 ```surql
 -- Define the parameter
-LET $name = "tobie";
+LET $surname = "Morgan Hitchcock";
 -- Use the parameter
-CREATE person SET name = $name;
+CREATE person SET name = "Tobie " + $surname;
```

Comment:
thought: maybe use `string::join` function?

Reply:
Good question. When I see `string::join` in a code sample the impression I get is maybe this language can't even use + to concatenate strings and needs long functions to do some pretty basic tasks...but then again when I see + then my impression is that maybe this language is loosey goosey like Javascript.

Reply:
I've opted for + in the first and string::join in the second with a tiny note. I debated doing this as it's unrelated to the subject at hand (the parameters) but is small enough that I think it ends up being a bit instructive without being distracting.

Reply:
Fair enough